bundles/org.openhab.binding.mercedesme/README.md

```diff
@@ -26,7 +15,7 @@ First time users shall follow the following sequence
 
 The Mercedes Me binding is based on the API of the Smartphone App.
 You have an account which is associated to one or more vehicles.
-Setup the Mercedes Me Account Bridge with your email address.
+Setup the Mercedes Me Account Bridge with your email address and password.
```

Comment:
```suggestion
Setup the Mercedes Me Account Bridge with your e-mail address and password.
```

Reply:
Really :)
I'm now briefed from my last binding mspa by @lsiepel to use email, not EMail or something like that

Reply:
I stand corrected: https://www.grammarly.com/blog/emailing/spelling-e-mail-email/. Apparently both are correct, and I'm too old-fashioned to have really noticed the "new" form without hyphen becoming accepted. 😄